### PR TITLE
add instance method "<<" to Spreadsheet::Worksheet

### PR DESCRIPTION
**Pull request by tetsuya-ogawa**
Add instance method "<<" to Spreadsheet::Worksheet for duck typing with CSV library.

### CODE REVIEW

**Comment by zdavatz** [comment]
Thank you for the PR, @tetsuya-ogawa - can you make the tests pass please for 1.8.7 and 1.9.2 as well? See [Build Jobs](https://travis-ci.org/zdavatz/spreadsheet/builds/640307337?utm_source=github_status&utm_medium=notification)

**Comment by tetsuya-ogawa** [comment]
@zdavatz 
Thank you for reply! Tests are passed.
Please check this PR.

**Comment by zdavatz** [comment]
merged. I will now release a new version.

**Comment by tetsuya-ogawa** [comment]
Thanks!

**Comment by zdavatz** [comment]
New release 1.2.6 is out.

File: lib/spreadsheet/worksheet.rb [hunk 1]
```diff
@@ -232,6 +232,9 @@ def insert_row idx, cells=[]
       updated_from idx
       res
     end
+    def << cells=[]
+      insert_row last_row_index + 1, cells
```
Comment (manxingxing) [inline lib/spreadsheet/worksheet.rb:236]: it fails when adding the first row, because `last_row_index` returns 0 when sheet is empty.

Comment (manxingxing) [reply lib/spreadsheet/worksheet.rb:236]:
```ruby
insert_row rows.size, cells
```

**Pull request merged**